.github/workflows/ci.yml

```diff
@@ -11,6 +11,9 @@ jobs:
 
       - name: Install dependencies
         run: pip install flake8
+      
+      - name: flake8 all repo only show erros
+        run: flake8 . || true # preventing CI failing while flake8 issues exist
 
-      - name: flake8
-        run: flake8 . || true # preventing CI failing while flake8 issues exist
\ No newline at end of file
+      - name: flake8 validating correct file(s)
+        run: flake8 asv/step_detect.py asv/runner.py asv/results.py asv/repo.py asv/main.py
```

Comment:
Are all the files in this PR added here? Seems like some are missing, is there a reason for it?

Reply:
Thank you, Marc. You are right! I'm going to include all the respective files.